ge04_team04/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Motorcycle Registry : Customer Discount",
+    "summary": "Give a $2500 discount to new customers",
+    "description": """ 
+    Motorcycle Registry : Customer Discount \n
+    This module allows you to give a $2500 discount to new customers purchasing an electric motorcycle for the first time.
+    Task id: 3426895 \n
+    Dev: team4 """,
+    "version": "0.1",
```

Comment:
Use Semantic Versioning
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes(I have never seen this number modified in odoo hehe)
MINOR version when you add functionality in a backward compatible manner(In odoo we increment this number when we upgrade code form versions)
PATCH version when you make backward compatible bug fixes(bug fixes in odoo as well)

So in this case, since the module is new version should be 1.0.0, but it's important to know when we increased it